### ISSUE TITLE
String value cannot be blank

### ISSUE THREAD

**Issue by jchristman** [martinandert/babel-plugin-css-in-js#10]
On [this line](https://github.com/martinandert/babel-plugin-css-in-js/blob/master/src/transformObjectExpressionIntoStyleSheetObject.js#L52) of transformObjectExpressionIntoStyleSheetObject, the string value is checked to see if it is blank, which is currently matching [all whitespace characters](https://github.com/martinandert/babel-plugin-css-in-js/blob/master/src/transformObjectExpressionIntoStyleSheetObject.js#L7).

This disallows CSS that looks like this:
```jsx
const stylesheet = cssInJS({
    default: {
        ':after': {
            content: " "    // There is a an empty space here so that the pseudo element renders
        }
    }
});
```

This is needed to do some styling things where content is not necessary for the pseudo element. A change that would fix this problem is simply:

```js
    assert(val.length >= 1, 'string value cannot be blank');
```

instead of the isBlank call. This still prevents empty strings, but allows deliberately white space strings for CSS properties.

**Comment by martinandert** [comment]
@jchristman Thanks for spotting!

This issue has been addressed in v1.2.3

**Comment by jchristman** [comment]
Excellent - thanks for the quick fix!
